src/App.css

```diff
@@ -1,39 +1,7 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  background-color: hsl(0, 0%, 100%);
```

Comment:
- [ ] please make sure to match the design requirements set by Microverse as close as possible.
your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/226752765-d3badc76-603b-404c-a2c3-f2abf33665c9.png)
Requested design:
![afbeelding](https://user-images.githubusercontent.com/39240820/226752830-b96d3f81-1a96-452c-9bbc-d28761e1aae1.png)